go.mod

```
@@ -31,6 +28,11 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require (
+	github.com/fsnotify/fsnotify v1.9.0 // indirect
+	go.opencensus.io v0.24.0 // indirect
+)
+
 require (
```

I do not think we should have more than one "indirect" block of dependencies. I would suggest moving these 2 indirect imports into the same block as the other block of indirect imports and then run again `go mod tidy` to properly sort them
```suggestion
require (
	github.com/fsnotify/fsnotify v1.9.0 // indirect
	go.opencensus.io v0.24.0 // indirect
```